build/AzurePipelinesTemplates/WindowsAppSDK-BuildProject-Steps.yml

```diff
@@ -4,8 +4,34 @@ parameters:
   buildOutputDir: '$(Build.SourcesDirectory)\BuildOutput'
   artifactName: 'drop'
   channel: 'experimental'
+  enableLicenseInstall: false 
+  # lincense installation assets are only accessible in internal builds, but are only 
+  # needed in release-signed stable builds. Only enble this functionality in the private 
+  # pipeline builds. Otherwise, defautl to an inert placeholder implementation.
```

Comment:
```suggestion
  # License installation assets are only accessible in internal builds, but are only 
  # needed in release-signed stable builds. Only enable this functionality in the private 
  # pipeline builds. Otherwise, default to an inert placeholder implementation.
```